pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -155,4 +158,7 @@ type HTTPSDConfig struct {
 	// Authorization header configuration to authenticate against the target HTTP endpoint.
 	// +optional
 	Authorization *v1.SafeAuthorization `json:"authorization,omitempty"`
+	// TLS configuration to use on every scrape request
```

Comment:
(nit) clarifying that this configuration doesn't apply to the scraped targets.
```suggestion
	// TLS configuration applying to the target HTTP endpoint.
```